lang/csharp/src/apache/main/Reflect/ArrayHelper.cs

```diff
@@ -0,0 +1,98 @@
+/*  Copyright 2019 Pitney Bowes Inc.
```

Comment:
According to [SOURCE FILE HEADERS FOR CODE DEVELOPED AT THE ASF](https://www.apache.org/legal/src-headers.html#headers), "there should be no copyright notice in the header". I believe these Copyright notices need to be removed. @cutting , @busbey, @Fokko  and/or others please correct me if I'm wrong.

Reply:
Hi @pa009fa ,
As @blachniet  has mentioned can you please cleanup the copyright on the header of the files?
Is there a reason you have them placed here or were those just the default ones? If there is a reason for these to be present please let us know the reason and we can look at it.

Thanks

Reply:
Sorry for delay. Piney Bowes has not contributed to an Apache project before so the legal team is getting an undertanding of how it works. As of this morning I got approval to remove the copyright from the source files and to add something to Notice.txt. I have made the changes and pushed to the branch.